### PR TITLE
Test change to README.md

### PR DESCRIPTION
**Pull request by rnathuji**
This is a test PR to confirm code owners behavior with branch protection rules.

### CODE REVIEW

**Review by MReyna12 — DISMISSED** [review]

**Comment by MReyna12** [comment]
Appears to work as intended for dev code owners.

**Comment by rnathuji** [comment]
@MReyna12 - Oh, actually, I don't think it was going to let me merge until you approved. I dismissed your review just to confirm / set it back to try and see if there's a config thing I missed.

**Pull request closed without merging**
